### PR TITLE
33 brand

### LINKED ISSUE #33: Component - Brand
**Widget**: Brand

**Image**:
Brand

**Dimensions/Styles**: https://zpl.io/bl0d6y0

**Inputs**: N/A

**Notes**:
- Fixed content, just a widget to be reused

### PR DESCRIPTION
**Pull request by CodeGlitch**
Widget proposal for brand

closes #33

### CODE REVIEW

File: lib/ui/widgets/brand.dart [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return RichText(
+      text: TextSpan(
+        children: <TextSpan>[
+          TextSpan(
+            text: '#COVID19',
+            style: TextStyle(
+              fontWeight: FontWeight.bold,
+              color: Colors.red,
```
Comment (danidroid) [inline lib/ui/widgets/brand.dart]: @CodeGlitch you need to update the color to use: Covid19Colors.red

File: lib/ui/widgets/brand.dart [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+              fontSize: 24.0,
+            ),
+          ),
+          TextSpan(
+            text: 'PT',
+            style: TextStyle(
+              fontWeight: FontWeight.bold,
+              color: Colors.green,
```
Comment (danidroid) [inline lib/ui/widgets/brand.dart]: same here: Covid19Colors.green

Comment (CodeGlitch) [reply lib/ui/widgets/brand.dart]: solved in 1dc79c9

File: lib/ui/widgets/brand.dart (hunk 1 above)
Comment (CodeGlitch) [reply lib/ui/widgets/brand.dart]: solved in 1dc79c9

File: lib/ui/widgets/brand.dart [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+class Brand extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return RichText(
+      text: TextSpan(
+        children: <TextSpan>[
+          TextSpan(
+            text: '#COVID19',
+            style: TextStyle(
+              fontWeight: FontWeight.bold,
```
Comment (miguelpruivo) [inline lib/ui/widgets/brand.dart]: I believe this should come from theme.

File: lib/ui/widgets/brand.dart [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+              color: Covid19Colors.red,
+              fontSize: 24.0,
+            ),
+          ),
+          TextSpan(
+            text: 'PT',
+            style: TextStyle(
+              fontWeight: FontWeight.bold,
```
Comment (miguelpruivo) [inline lib/ui/widgets/brand.dart]: This as well. 👍

Comment (CodeGlitch) [reply lib/ui/widgets/brand.dart]: removed on 661791e

File: lib/ui/widgets/brand.dart (hunk 3 above)
Comment (CodeGlitch) [reply lib/ui/widgets/brand.dart]: removed on 661791e

**Pull request merged**